app/controllers/spree/custom_checkout_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Spree
```

Comment:
why we need this new controller. Any specific reason?

Reply:
it was due to refresh the cart drawer